tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+User clicks on 'View preview token log' link inside the 'Draft version details' section
+    user clicks link by index    View changelog and guidance notes    2
+
+Validate the contents in the 'API dataset changelog' page.
+    user waits until page contains    API data set changelog
+
+    user waits until page contains    Content for the public guidance notes
+    user clicks link    Back to API data set details
+
+Add headline text block to Content page for the second release
+    user navigates to content page    ${PUBLICATION_NAME}
+    user adds headlines text block
+    user adds content to headlines text block    Headline text block text
+
+Approve second release
+    user clicks link    Sign off
+    user approves release for immediate publication
+
+Verify newly published release is on Find Statistics page
+    user checks publication is on find statistics page    ${PUBLICATION_NAME}
+
+User navigates to data catalogue page
+    user navigates to data catalogue page on public frontend
+
+Search with 2nd API dataset
+    user clicks element    id:searchForm-search
+    user presses keys    ${PUBLICATION_NAME}
+    user clicks radio    API data sets only
+
+    user waits until page finishes loading
+    user clicks radio    Newest
+
+    user checks summary list contains    Status    This is the latest data
+    ...    parent=testid:data-set-file-summary-${SUBJECT_2_NAME}
+    user checks summary list contains    Status    Available by API
+    ...    parent=testid:data-set-file-summary-${SUBJECT_2_NAME}
+    user checks page contains link    ${SUBJECT_2_NAME}
+
+User clicks on 2nd API dataset link
+    user clicks link    ${SUBJECT_2_NAME}
+    user waits until page finishes loading
+    user waits until h1 is visible    ${SUBJECT_2_NAME}
+
+User checks relevant headings exist on API dataset details page
+    user waits until h2 is visible    Data set details
+    user waits until h2 is visible    Data set preview
+    user waits until h2 is visible    Variables in this data set
+    user waits until h2 is visible    Using this data
+    user waits until h2 is visible    API data set quick start
+    user waits until h2 is visible    API data set version history
+    user waits until h2 is visible    API data set changelog
+
+User verifies the public data guidance in the 'API data set changelog' section
+    user waits until element contains    testid:public-guidance-notes    Content for the public guidance notes
+
+User verifies major changes in the 'API data set changelog' section
+    user waits until h3 is visible    Major changes for version 2.0
+    ${major_changes_Section}=    get child element    id:apiChangelog    testid:major-changes
+
+    ${school_types_filter}=    user checks changelog section contains deleted filter    ${major_changes_Section}
```

Comment:
.Uppercased `Section` - should be lowercased `section`

Reply:
Got it ta